packages/storage-plus/src/prefix.rs

```diff
@@ -153,6 +156,96 @@ where
     }
 }
 
+#[derive(Clone)]
+pub struct Prefix2<K, T>
```

Comment:
Could this name be changed to something more descriptive?

Reply:
See #439. Will rebase / merge these recent changes into it, for ease of reviewing.